FallHandling.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using BattleTech;
+
+namespace CharlesB
+{
+    public class FallHandling
```

Comment:
Thinking about naming, I would probably flip the name of the class and the function here to be more like `Piloting.HandleFall.Say()`.